openedx/core/djangoapps/ccxcon/migrations/0002_auto_20160325_0407.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('ccxcon', '0001_initial_ccxcon_model'),
+    ]
+
+    operations = [
+        migrations.AlterModelOptions(
+            name='ccxcon',
+            options={'verbose_name': 'CCX Connector', 'verbose_name_plural': 'CCX Connectors'},
```

Comment:
@giocalitri You seems to be a relevant person for ccxcon, there was a missing migration for ccxcon and i've added it back, does this make sense to you ?